src/aws/Makefile

```diff
@@ -16,8 +19,11 @@ refresh: ## Refresh Terraform config
 refresh:
 	docker-compose run --rm app /bin/terraform refresh
 
+env.d/state_bucket:
+	cp env.d/state_bucket.dist env.d/state_bucket
+
 .PHONY: state-create
-state-create: ## Create bucket in AWS to store Terraform state files
+state-create: env.d/state_bucket ## Create bucket in AWS to store Terraform state files
```

Comment:
`env.d/development` is needed here but not `env.d/state_bucket`. In fact the bucket creation is not done correctly because the aws region is hardcoded (in `create_state_bucket/variables.tf`), the region defined will not be used. An issue should be open to fix this issue.

Reply:
Then I think the state_bucket environment file can be removed since it is defining variables that are also defined in env.d/development. It's also better because it ensures coherence between the 2 Terraform projects.

On your comment that it is hardcoded, my understanding is that a default value is given but that it can be overriden by setting the `TF_VAR_aws_region` (which is the case in env.d/development.dist...).